Add configuration to bubble group targets

feliciahalim opened the pull request:
by adding -output-dir and output tree in SVG format. Moreover, in the `Makefile.commo`n, I have added `BUBBLE_TARGETS`in the clean configuration directory list.

feliciahalim committed:
Add configuration to bubble group targets by adding -output-dir and output the tree in SVG file

feliciahalim committed:
Modify clean configuration by adding BUBBLE_TARGETS in the list of removed directory

domainexpert commented:
@feliciahalim Instead of adding `BUBBLE_TARGETS` to in `Makefile.common`'s `clean` target, I suggest adding it to `EXTRA_REMOVAL=` in `scalability/Makefile`, such that `BUBBLE_TARGETS` is localized within `scalability/Makefile`. That is, one should instead modify `EXTRA_REMOVAL=${TR_TARGETS}` in `scalability/Makefile` to `EXTRA_REMOVAL=${TR_TARGETS} ${ECHO_TARGETS} ${BUBBLE_TARGETS}`.

feliciahalim committed:
Move BUBBLE_TARGETS from Makefile.common into scalability/Makefile such that BUBBLE_TARGETS is localized within scalability/Makefile

feliciahalim committed:
Add -interpolation-stat option in the TR special targets group

feliciahalim commented:
@domainexpert Thank you very much for the suggestion. I have fixed the configuration and also added -`interpolation-stat `option when executing `TR` special targets group.
